### PR TITLE
Update code to near perfection

### PR DESCRIPTION
**Pull request by PrettyPearl2302**
## Description
In this pull request, I update my code in several ways. The main purpose id to make sure is clean and near perfect. 

1. I seeded the comments database, using faker.js
2. I debugged ratings by switching it back to using a styled component as opposed to CSS
3. I debugged login by not re-running updatePassword.js, and rewriting the code to compare passwords to brcypt.compare. Basically, it wasn't really broken.
4. I took out the Sidebar component from app.jsx, and imported it separately in the different components. This is because importing in app.jsx made it show up even on the login/sign up page
5. I took comments
6. I took out remaining console logs
7. I deleted components and files that were useless e.g empty CSS files, and Footer.jsx because I realized that I didn't necessarily need a footer

## Milestones
PROJECT COMPLETION

## Resources
https://v6.fakerjs.dev/api/datatype.html#string

## Test Plan
There is not necessarily a test plan here because most of the things I implemented have either being done before or don't necessarily affect the display

### CODE REVIEW

File: timify-api/route/users.js [hunk 1]
```diff
@@ -51,6 +49,8 @@ router.post('/users/login', async (req, res) => {
 
     req.session.user = user
 
+    console.log(user)
```
Comment (ghost) [inline timify-api/route/users.js:52]: nit: Should remove user info here.

**Review by ghost — APPROVED** [review]
Overall looks good!

File: timify-api/server.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ import SequelizeStoreInit from 'connect-session-sequelize'
 
 const app = express()
 
-const sessionTime = 365 * 24 * 60 * 60 * 1000 // 1 year in milliseconds
+const sessionTime = 365 * 24 * 60 * 60 * 1000
```
Comment (ushanakg) [inline timify-api/server.js:16]: I think if you're going to remove this comment you should use the variable name to make it clear this is 1 year. the goal is readability

File: timify-ui/src/components/Rating/Rating.jsx [hunk 3]
```diff
@@ -58,30 +58,30 @@ const Rate = ({episodeId}) => {
...
-						</div>
-						<div className="rating">
+						/>
+						
+							<Rating>
 							<FaStar
 								color={
 									givenRating < rate || givenRating === rate
 										? "ffae42"
 										: "rgb(192,192,192)"
```
Comment (ushanakg) [inline timify-ui/src/components/Rating/Rating.jsx:77]: is it possible to save these colors as variables somehow so the names are more descriptive to us?

**Review by ushanakg — APPROVED** [review]

**Pull request merged**